### PR TITLE
docs(notifications): add notifications control guide

### LINKED ISSUE #46: Add notifications control guide

### PR DESCRIPTION
**Pull request by andrewazores**
Fixes #46
Depends on #37

### CODE REVIEW

File: guides/_subsections/configure-graphical-notifications.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+## [Configure Graphical Notifications](#configure-graphical-notifications)
+Actions such as starting/stopping/archiving recordings and state changes such as running recordings stopping produce
```
Comment (jan-law) [inline guides/_subsections/configure-graphical-notifications.md:2]: "running recordings stopping" refers to recordings that stop after a fixed duration, right? For clarity, what do you think of writing it as "recordings stopping after a fixed duration" to distinguish it from manual recording stops?

Comment (andrewazores) [reply guides/_subsections/configure-graphical-notifications.md:2]: That sounds good. Now it's making me wonder if we should be emitting a notification on manual stop, too.

Comment (jan-law) [reply guides/_subsections/configure-graphical-notifications.md]: > Now it's making me wonder if we should be emitting a notification on manual stop, too.

Aren't we already emitting notifications on manual stop?

Comment (andrewazores) [reply guides/_subsections/configure-graphical-notifications.md]: I don't know. Probably? It sounds like something we would be doing... but I haven't checked :-)

**Review by hareetd — APPROVED** [review]
Looks good.

**Pull request merged**